WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -445,35 +445,38 @@ private extension DashboardViewModel {
                               canShowAnalytics: Bool) async {
         dashboardCards = await {
             if let stored = await loadDashboardCards() {
-                return stored
+                var mutableStored = stored
+                // Remove saved analytics cards if they should not be shown anymore.
+                if !canShowAnalytics {
+                    mutableStored.removeAll { $0.type == .performance || $0.type == .topPerformers }
+                }
```

Comment:
It looks like this logic is different from the default case below. Should we have the same solution, i.e. removing the stats cards or disabling them for both cases?

Reply:
Now that I think again this makes sense to me. Maybe it's clearer if we do rewrite this a bit:

```Swift
let analyticCardTypes: [DashboardCard.CardType] = [.performance, .topPerformers]
// Only display analytics cards if allowed.
return canShowAnalytics ? stored : stored.filter { analyticCardTypes.contains($0) == false }
```